packages/ecs-agent/ecs-tmpfiles.conf

```
@@ -1,2 +1,4 @@
 d /var/lib/ecs/data 0700 root root
 d /var/log/ecs 0755 root root
+d /var/log/ecs/exec 0755 root root -
+d /var/ecs/managed-agents/execute-command/config 0750 root root -
```

If we don't need this state preserved across updates, it might be better to clean up the directory. That way, if we're ever downgrading into this version from a newer version, we won't confuse the ecs-agent with data that it's not able to handle.

```suggestion
d /var/log/ecs/exec 0755 root root -
R /var/ecs/managed-agents - - - -
d /var/ecs/managed-agents/execute-command/config 0750 root root -
```